os/fs/driver/mtd/smart.c

```diff
@@ -493,73 +493,6 @@ static void smart_free(FAR struct smart_struct_s *dev, FAR void *ptr)
 }
 #endif
 
-/***************************************************************************************
- *   This is called from the fs_clean() API to release the memory acquired by MTD driver
- **************************************************************************************/
-int smart_clean(FAR struct inode *inode)
```

Comment:
OK. I think it's better to release memory in mops something like that mops->unbind...
As you know smart_clean(), fs_clean() cannot be called in user application because it directly calls kernel space functions. 
Please, re-implement those kinds of API functions in user space.
Thanks.